ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/State/Types.hs

```diff
@@ -109,13 +110,15 @@ sequenceHardForkState (HardForkState tel) =
   Supporting types
 -------------------------------------------------------------------------------}
 
--- | Translate @f x@ to @f y@ across an era transition
+-- | Translate @f x@ to @g y@ across an era transition
 --
--- Typically @f@ will be 'LedgerState' or 'WrapChainDepState'.
-newtype Translate f x y = Translate {
-      translateWith :: EpochNo -> f x -> f y
+-- Typically @f@/@g@ will be 'LedgerState' or 'WrapChainDepState'.
+newtype Translate' f g x y = Translate {
```

Comment:
Could we hardcode `Ticked :.: f` instead of introducing `g`?

Reply:
If so, perhaps call it `TranslateAndUntick`?


... although ... will the translation logic ever make use of the extra info in the `Ticked` wrapper? I'd need to look and think more carefully

Reply:
Nice, just pushed, should be addressed there :smile: